replicatedhq/ship
# load resources from state into UI tree

What I Did
------------

Load `Resources` from state.json into the UI tree alongside patched


How I Did it
------------

Copy paste a lot of code around patches in `LoadTree`. There's some dupe emerging, but I feel that for now its the right kind of duplication, I dont think  we ought to be trying to consolidate an abstraction around patches vs resources.


How to verify it
------------

```
ship init ...
vim state.json # add resources
ship update --headed # see resources in tree
```


Description for the Changelog
------------

Load resources from state into UI tree
